### PR TITLE
Partial add sites from GroobyNetwork

### PR DESCRIPTION
**Pull request by ZZenvo**
Not all GroobyNetwork sites are working with this scraper, but here are the ones that does seem to. Further comments in the scraper .yml

### CODE REVIEW

File: scrapers/GroobyNetwork-Partial.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      - transgasm.com #works
+      # - ts-castingcouch.com
+      # - uk-tgirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //p[@class="trailertitle"]/text() # done
+      Date:
+        selector: //div[@class="setdesc"]/text() # done
```
Comment (bnkai) [inline scrapers/GroobyNetwork-Partial.yml:50]: the date selector  returns some extra stuff
switching to 
```yaml
selector: //div[@class="setdesc"]//b[contains(.,"Added")]/following-sibling::text()[1]
```
works even for `grooby-archives.com`

File: scrapers/GroobyNetwork-Partial.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        parseDate: January 2, 2006
+      Details: //div[@class="trailerpage_info"]/p[2]/text() # done
```
Comment (bnkai) [inline scrapers/GroobyNetwork-Partial.yml:52]: probably safer for details 
```yaml
Details: //div[@class="trailerpage_info"]/p[not(@class)]/text()
```

File: scrapers/GroobyNetwork-Partial.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      Performers:
+        Name: //div[@class="setdesc"]//a/text() # done
+      Studio:
+        Name: //meta[@name="author"]/@content # done
+      Image:
```
Comment (bnkai) [inline scrapers/GroobyNetwork-Partial.yml:57]: in some sites the content for the image isn't a valid URL eg in bobstgirls
```yaml
      Image:
        selector: //meta[@property="og:image"]/@content # done
        postProcess:
          - replace:
            - regex: ^// # bobstgirls
              with: "https://"
```
takes care of that

**Comment by bnkai** [comment]
Tested half of the "working" sites. Will update when i test the rest.

**Comment by ZZenvo** [comment]
Okay. Again, thanks for going through all these! Learning a lot along the way... xpath is quite powerful:)

**Comment by bnkai** [comment]
@ZZenvo  checked out ok the rest of the "working" sites.
Can you do a cleanup removing the unneeded comments and the not working sites ?

**Comment by ZZenvo** [comment]
@bnkai Okay, cool. Will update with your feedback and do a cleanup when possible.

**Pull request merged**